genisys/server/external/imports/ui/App.js

```diff
@@ -0,0 +1,9 @@
+import { Template } from 'meteor/templating';
+import { TasksCollection } from "../api/TasksCollection";
+import './App.html';
+
+Template.mainContainer.helpers({
+  tasks() {
+    return TasksCollection.find({}).fetch();
```

Comment:
@wreiske Do you know why this doesn't seem to be querying the DB? I've followed the official Blaze tutorial and can't seem tog et it to work. Putting a console.log statement in this function to see what is being returned shows an empty array. The results are also the same when just using just `TasksCollection.find({})`

Reply:
I should also add that I've checked the database manually through Mongosh and confirmed that the collection exists and has data associated with it.

Reply:
I actually solved this in [5cf2160](https://github.com/xeluior/genisys/pull/79/commits/5cf2160ea5f23aa2caf125ace796fab1524cb634), I believe that the Meteor tutorial I was following is out of date and assumed that the autopublish package was installed.

Reply:
Check out https://packosphere.com/ostrio/flow-router-extra and make sure you're publishing the data server side.

Have you gone through https://blaze-tutorial.meteor.com/ ? Check out https://blaze-tutorial.meteor.com/simple-todos/09-publications

Reply:
https://github.com/veliovgroup/flow-router/blob/master/docs/quick-start.md is a great start for FlowRouter, since it shows you how to waitOn for a subscription.